skbio/alignment/tests/test_tabular_msa.py

```diff
@@ -23,9 +23,9 @@
 from skbio import Sequence, DNA, RNA, Protein, TabularMSA
 from skbio.sequence import GrammaredSequence
 from skbio.util._decorator import classproperty, overrides
-from skbio.util._testing import (ReallyEqualMixin, MetadataMixinTests,
-                                 PositionalMetadataMixinTests,
-                                 assert_index_equal,
+from skbio.metadata._testing import (ReallyEqualMixin, MetadataMixinTests,
```

Comment:
`ReallyEqualMixin` should stay at its original location (`skbio.util_testing`) because it is a general testing utility.